samples/go/blob-get/blob_get.go

```diff
@@ -51,24 +51,32 @@ func main() {
 	}
 
 	filePath := flag.Arg(1)
+	var file *os.File
+	showProgress := true
 	if filePath == "" {
-		d.CheckErrorNoUsage(errors.New("Empty file path"))
+		file = os.Stdout
```

Comment:
Though I think you'd only want to do this if stdout is TTY. It's fun to be able to go `blob-get foobar::mypath > file.txt`.